system-addon/lib/SectionsManager.jsm

```diff
@@ -61,18 +61,22 @@ const BUILT_IN_SECTIONS = {
     icon: "highlights",
     title: {id: "header_highlights"},
     maxRows: 3,
-    availableLinkMenuOptions: ["CheckBookmarkOrArchive", "CheckSavedToPocket", "Separator", "OpenInNewWindow", "OpenInPrivateWindow", "Separator", "BlockUrl", "CheckDeleteHistoryOrEmpty"],
     emptyState: {
       message: {id: "highlights_empty_state"},
       icon: "highlights"
     },
     shouldSendImpressionStats: false
   })
 };
+const CONTEXT_MENU_OPTIONS_FOR_HIGHLIGHT_TYPES = {
+  history: ["CheckBookmark", "CheckSavedToPocket", "Separator", "OpenInNewWindow", "OpenInPrivateWindow", "Separator", "BlockUrl", "DeleteUrl"],
```

Comment:
personally I'd like a test ensuring all of the possible highlight types that get produced by highlights feed match up with these to handle possible future breakages... but that might be hard to do?

Reply:
although that's sounding more like an integration test

Reply:
could write a mochitest for it as a follow up?